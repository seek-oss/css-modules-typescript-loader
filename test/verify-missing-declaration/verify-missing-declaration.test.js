const compiler = require('../compiler.js');
const getErrorMessage = require('../getErrorMessage');

test('Can error on invalid declaration', async () => {
  expect.assertions(1);

  try {
    await compiler(require.resolve('./index.js'), {
      mode: 'verify'
    });
  } catch (err) {
    expect(getErrorMessage(err.errors[0])).toMatchSnapshot();
  }
});
