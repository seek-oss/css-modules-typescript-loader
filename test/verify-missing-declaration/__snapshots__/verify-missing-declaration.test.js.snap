// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Can error on invalid declaration 1`] = `
"Error: Generated type declaration does not exist. Run webpack and commit the type declaration for '/test/verify-missing-declaration/index.css.d.ts'
"
`;
